modules/ROOT/pages/6.7-release-notes.adoc

```diff
@@ -279,7 +279,13 @@ For information on using premium skins and icon packs, see: xref:premium-skins-a
 
 === If loading content CSS takes more than 500ms, the editor will be set to an *in progress* state until the CSS is ready
 // TINY-10008
+Previously, an issue was identified that was reproducible during the {productname} initialization process.
 
+If the `contentEditable: 'true'` property was set within the editors configuration, and the user host browser was affected by a slow internet connection such as "3G", and the `contentCss` was still loading, the user could interact with the editor before the `ContentCss` had finished its loading phase.
+
+As a consequence, after the `ContentCss` had completed its initial load, any content inserted into the editor before the `ContentCss` had completed loading was removed from the editor.
+
+{productname} 6.7 addresses this issues, which now blocks any interaction with the editor while the `contentCss` is loading. As a result of this fix, it is no longer possible to insert any text into the editor during the `contentCss` loading phase.
```

Comment:
```suggestion
If a {productname} configuration included a `contentEditable: 'true'` setting (or had no explicit global setting for `contentEditable`, and therefore defaulted this option to `'true'`) and the host browser was operating over a relatively slow connection, it was possible to add content to the editor before `ContentCss` finished loading.

This could lead to data loss. After `ContentCSS` completed its initial load, any content added to the editor before the load had completed was removed from the editor.

{productname} 6.7 addresses this by blocking interaction with the editor until `ContentCss` has completed loading, disallowing content insertion until initialisation is complete.

A relatively slow network connection for this purpose is an ≅ 400 kb/sec connection with an ≅ 2000 ms latency; equivalent to a Slow 3G wireless connection. The potential UX change caused by this change will, therefore, be experienced by very few users.
```
Suggested edit.